boilerplate/_lib/common.sh

```diff
@@ -53,6 +57,30 @@ if [ -z "$BOILERPLATE_GIT_CLONE" ]; then
   export BOILERPLATE_GIT_CLONE="git clone $BOILERPLATE_GIT_REPO"
 fi
 
+## Information about the boilerplate consumer
+# E.g. "openshift/my-wizbang-operator"
+CONSUMER=$(repo_name .)
+[[ -z "$CONSUMER" ]] && err "
+Failed to determine current repository name"
+#
```

Comment:
What's up with the empty comment lines?

Reply:
Just a way to tie this section (starting on L60) together visually.